pkg/upload/dsym.go

```diff
@@ -111,9 +110,26 @@ func ProcessDsym(
 				}
 			}
 
-		} else if dsymPath == "" {
-			log.Info("No Xcode project, workspace or package in '" + path + "'")
+		} else {
 			dsymPath = path
+
+			if xcodeProjPath != "" {
+				if scheme == "" {
+					var err error
+					scheme, err = ios.GetDefaultScheme(xcodeProjPath)
+					if err != nil {
+						log.Warn(err.Error())
+					}
+
+					if scheme != "" {
+						var err error
+						buildSettings, err = ios.GetXcodeBuildSettings(xcodeProjPath, scheme)
+						if err != nil {
+							return err
+						}
+					}
+				}
+			}
```

Comment:
To avoid this duplication, could we do the following before the if-statement on L67:

```
if ios.IsPathAnXcodeProjectOrWorkspace(path) {
    if xcodeProjPath == ""  {
        xcodeProjPath = path
    }
} else {
    dsymPath = path
}

if xcodeProjPath != "" {
    // As above
} 
```